scripts/install-windows-prereqs.ps1

```diff
@@ -516,23 +529,38 @@ function Install-DCAP-Dependencies {
     Install-Tool -InstallerPath $PACKAGES["dcap"]["local_file"] `
                  -ArgumentList @('/auto', "$PACKAGES_DIRECTORY\Intel_SGX_DCAP")
 
+    $OS_VERSION = Get-WindowsRelease
     if (($LaunchConfiguration -eq "SGX1FLC") -or ($LaunchConfiguration -eq "SGX1FLC-NoDriver") -or ($DCAPClientType -eq "Azure"))
     {
         $drivers = @{
-            'sgx_base_dev' = @{
-                'zip_path'    = "$PACKAGES_DIRECTORY\Intel_SGX_DCAP\Intel SGX DCAP for Windows *\LC_driver_WinServer2016\Signed_*.zip"
-                'location'    = 'root\SgxLCDevice'
-                'description' = 'Intel(R) Software Guard Extensions Launch Configuration Service'
+            'WinServer2016' = @{
+                'sgx_base_dev' = @{
+                    'zip_path'    = "$PACKAGES_DIRECTORY\Intel_SGX_DCAP\Intel SGX DCAP for Windows *\LC_driver_${OS_VERSION}\Signed_*.zip"
+                    'location'    = 'root\SgxLCDevice'
+                    'description' = 'Intel(R) Software Guard Extensions Launch Configuration Service'
+                }
+                'sgx_dcap_dev' = @{
+                    'zip_path'    = "$PACKAGES_DIRECTORY\Intel_SGX_DCAP\Intel SGX DCAP for Windows *\DCAP_INF\${OS_VERSION}\Signed_*.zip"
+                    'location'    = 'root\SgxLCDevice_DCAP'
+                    'description' = 'Intel(R) Software Guard Extensions DCAP Components Device'
+                }
             }
-            'sgx_dcap_dev' = @{
-                'zip_path'    = "$PACKAGES_DIRECTORY\Intel_SGX_DCAP\Intel SGX DCAP for Windows *\DCAP_INF\WinServer2016\Signed_*.zip"
-                'location'    = 'root\SgxLCDevice_DCAP'
-                'description' = 'Intel(R) Software Guard Extensions DCAP Components Device'
+            'WinServer2019' = @{
+                'sgx_base' = @{
+                    'zip_path'    = "$PACKAGES_DIRECTORY\Intel_SGX_DCAP\Intel SGX DCAP for Windows *\LC_driver_${OS_VERSION}\Signed_*.zip"
+                    'location'    = 'root\SgxLCDevice'
+                    'description' = 'Intel(R) Software Guard Extensions Launch Configuration Service'
+                }
+                'sgx_dcap' = @{
+                    'zip_path'    = "$PACKAGES_DIRECTORY\Intel_SGX_DCAP\Intel SGX DCAP for Windows *\DCAP_INF\${OS_VERSION}\Signed_*.zip"
+                    'location'    = 'root\SgxLCDevice_DCAP'
```

Comment:
DCAP is a Software Component device on 16299 and later, should be:

SWC\VEN_INT&DEV_0E0C_DCAP

Reply:
Line 556, instead of 'root\...'
it should be 'SWC\VEN_INT&DEV_0E0C_DCAP'